bpfd/src/command.rs

```diff
@@ -565,34 +495,48 @@ impl Program {
             Program::Tracepoint(p) => p.data.kernel_info = Some(info),
             Program::Kprobe(p) => p.data.kernel_info = Some(info),
             Program::Uprobe(p) => p.data.kernel_info = Some(info),
+            // has to be set at variant creation
```

Comment:
Is it a noop? Or an error?
Feels to me like it should probably be an error.

Reply:
decided to panic here instead of returning an error